cmu_graphics/cmu_graphics.py

```diff
@@ -340,6 +340,31 @@ def run(self):
         pygame.quit()
         os._exit(0)
 
+class AppWrapper(object):
```

Comment:
What's the purpose of this object?

Reply:
There's a lot of user-visible garbage in the app object like `_allKeysDown`, `_runningKeyMap`, `_msPassed`, etc. This hides those properties and ensures that they're not accidentally overwritten